src/simpla-text-editor.html

```diff
@@ -107,9 +205,10 @@
         this.is = 'simpla-text-toolbar';
 
         this.properties = {
+
           tools: {
             type: Object,
-            value: INITIAL_TOOLS
+            value: () => INITIAL_TOOLS
```

Comment:
Ohhh sorry I thought it was a typo, because every other time I've seen you do it this way. Okie dokie